lib/cisco_node_utils/bgp.rb

```diff
@@ -368,28 +353,19 @@ def default_confederation_peers
 
     # Graceful Restart Getters
     def graceful_restart
-      match = config_get('bgp', 'graceful_restart', @get_args)
-      match.nil? ? false : true
+      config_get('bgp', 'graceful_restart', @get_args)
```

Comment:
Note:  It looks like this change fixes a bug where matching 'no graceful-restart' returned true instead of false.  We should merge develop into feature/grpc after this PR is merged.

Yaml for reference:

```
graceful_restart:
  kind: boolean
  config_get_token_append: '/^(no )?graceful-restart$/'
  config_set_append: '<state> graceful-restart'
  default_value: true
```